charts/camunda-platform/templates/connectors/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.connectors.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "connectors.fullname" . }}
+  labels: {{- include "connectors.labels" . | nindent 4 }}
+  annotations: {{- toYaml .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "connectors.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "connectors.labels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- toYaml .Values.podLabels | nindent 8 }}
+        {{- end }}
+        {{- if .Values.podAnnotations }}
+        annotations: {{- toYaml  .Values.podAnnotations | nindent 8 }}
+        {{- end }}
+    spec:
+      imagePullSecrets:
+        {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: {{ include "camundaPlatform.image" . | quote }} # has to be camunda/connectors-bundle:0.16.0
+          imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+          {{- if .Values.containerSecurityContext }}
+          securityContext: {{- toYaml .Values.containerSecurityContext | nindent 10 }}
+          {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 10 }}
+          ports:
+            - containerPort: 8080
+          env:
+            - name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
+              value: "{{ .Release.Name }}-zeebe:26500" # Must be zeebe:26500
```

Comment:
@aabouzaid is there a better way to get a Zeebe DNS name with port?

Reply:
since now the connectors are part of the parent chart, you can access the vars directly from child charts.
so you can use the same as in `charts/camunda-platform/charts/zeebe-gateway/templates/gateway-service.yaml`

so it will be like this:
```
- name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
value: "{{ include "zeebe.names.gateway" . }}:{{ .Values.zeebe-gateway.service.gatewayPort }}"
```

you might have a problem accessing `.Values.zeebe-gateway` because it has dash it the name, and you could use the `index` Helm function to overcome it.